app/models/cqm/measure.rb

```diff
@@ -84,6 +86,9 @@ def as_json(*args)
       unless self.source_data_criteria.nil?  || !self.source_data_criteria.any?
         result['source_data_criteria'] = self.source_data_criteria.map{ |x| x.as_json(*args) }
       end
+      unless self.population_sets.nil?  || !self.population_sets.any?
```

Comment:
ideally, it should be reflected in transform_json as well

Reply:
Agreed. I will add it in another PR before closing the story since this PR is needed to unblock frontend work.